src/ui_dialogs.ts

```diff
@@ -115,17 +164,270 @@ export function openEditLinkDialog(column:LinkColumn, $header:d3.Selection<Colum
   });
 }
 
+
+// Renderer type change
+export function renderertypedialog(column: Column, $header: d3.Selection<Column>) {
+  var renderertype = (<any>column.desc).type;
+  var valuestring: any = (<any>column.desc).renderertype;
```

Comment:
why does the list of possible renderer types for each columns needs to be defined. e.g. if it is a number column, there are just two options: circle and bar. Why is this not stored in the model class itself?